libnavigation-router/src/test/resources/multi_leg_route_refreshed.json

```diff
@@ -4,7 +4,7 @@
       "weight_typical": 1511.56,
       "routeIndex": "0",
       "distance": 2839.016,
-      "duration": 615.8249999999998,
+      "duration": 726.8249999999998,
```

Comment:
The refresh response in this test case swaps the data in a waypoint to be a charger with charge time `111`.